modules/dnn/test/test_ie_models.cpp

```diff
@@ -108,6 +108,10 @@ static const std::map<std::string, OpenVINOModelTestCaseInfo>& getOpenVINOTestMo
             "intel/person-detection-retail-0013/FP32/person-detection-retail-0013",
             "intel/person-detection-retail-0013/FP16/person-detection-retail-0013"
         }},
+        { "age-gender-recognition-retail-0013", {
```

Comment:
Please update the "how to" comment above

Reply:
Done.